chat-server/tests/integrations/openai.test.ts

```diff
@@ -0,0 +1,43 @@
+import { Axios } from "axios";
+import { OpenAIClient } from "../../src/integrations/openai";
+import dotenv from "dotenv";
+dotenv.config();
```

Comment:
We might not want to be using that env file for a test. Is there a way to have dotenv prefer a ".env.test" if one exists?

Reply:
to the best of my understanding, there's not a simple way to do this. 

the easy way would be to have a separate `.env.test` file. for our purposes here, at least for now, all the values would be the same as the `.env` file, so i don't think we need to add.

Reply:
also while investigating this, learned we can replace these lines with: `import "dotenv/config";`

applied that throughout